tests/network/vmi_multus.go

```diff
@@ -576,6 +580,45 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
+		Context("kubectl get vmi test for primary and secondary interfaces", func() {
+			It("should verify vmi ip value match primary interface ip value", func() {
+				Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, "linux-br", bridge10CNIType, bridge10Name, 0, "", bridge10MacSpoofCheck)).To(Succeed())
```

Comment:
Please use constants instead of free strings.

Reply:
fixed